rotkehlchen/globaldb/handler.py

```diff
@@ -1245,9 +1246,26 @@ def get_historical_prices(
 
         prices_results = []
         with GlobalDBHandler().conn.read_ctx() as cursor:
-            for entry in querylist:
-                result = cursor.execute(querystr, entry).fetchone()  # below last index of the result tuple is ignored in deserialize  # noqa: E501
-                prices_results.append(None if result[0] is None else HistoricalPrice.deserialize_from_db(result))  # noqa: E501
+            for query_entry, (from_asset, to_asset, queried_timestamp) in zip(querylist, query_data, strict=True):  # noqa: E501
+                result = cursor.execute(querystr, query_entry).fetchone()  # below last index of the result tuple is ignored in deserialize  # noqa: E501
+                if result[0] is None:
+                    prices_results.append(HistoricalPrice(
```

Comment:
So have a problem with this. Zero price is one thing. Price not found (`None`) is another. And this distinction will be helpful in many places.

I don't think we shoiuld confuse those two.